modules/generating-icsp-object-scoped-to-a-registry.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// * updating/updating-restricted-network-cluster.adoc:
+
+[id="generating-icsp-object-scoped-to-a-registry_{context}"]
+= Widening the mirror registry scope of the ImageContentSourcePolicy object
+
+Use the `--icsp-scope=registry` flag with the `oc adm catalog mirror` command to widen the mirror registry scope in the `ImageContentSourcePolicy` object. This minimizes the number of times the cluster nodes reboot when updating a registry entry.
+
+The `oc adm catalog mirror` command generates an `ImageContentSourcePolicy` resource that maps the original container image repository to a local image catalog repository that you create and mirrors it. This typically occurs inside a disconnected environment. When you apply a new or modified `ImageContentSourcePolicy` resource to a cluster, the `oc adm catalog mirror` command generates a configuration file and places it onto each node. This configuration file is used by CRI-O, the container runtime that manages the containers running on the nodes. When you update the configuration file for a node, the node reboots. A widely scoped `ImageContentSourcePolicy` resource reduces the number of times the cluster nodes need to reboot in response to changes to the resource.
+
```

Comment:
Because the intro to the assembly covers a bit about the disconnected environment and the registry, my suggestion is to omit that content here because it obscures what the customer wants to accomplish--fewer reboots.  My $0.02:

> The `oc adm catalog mirror` command generates an `ImageContentSourcePolicy` resource that maps the original container image repository to a local image catalog repository.
> When you apply a new...

Nit: s/cluster nodes/nodes for consistency.

Reply:
Sounds good to me!